apps/src/templates/VerticalImageResourceCard.jsx

```diff
@@ -84,7 +86,7 @@ class VerticalImageResourceCard extends Component {
       <div style={cardStyle}>
         <div style={imageStyle}>
           <a href={link}>
-            <img src={imgSrc} alt={title} />
+            <img src={imgSrc} alt={altText} />
```

Comment:
This is a behavior change. I think that's okay, as the `title` is read within this component anyways.

Reply:
Agreed. A net positive behavior change.